__tests__/integration.test.js

```diff
@@ -107,6 +107,34 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200: returns an array of article objects sorted by a valid column name in descending order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=article_id")
+      .expect(200)
+      .then(({ body: { articles } }) => {        
+        expect(articles).toBeSortedBy("article_id", {
+          descending: true,
+        });
+      });
+  });
+  test("200: returns an array of article objects sorted by a valid column name in ascending order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=comment_count&order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {                
+        expect(articles).toBeSortedBy("comment_count", {
+          ascending: true,
+        });
+      });
+  });
```

Comment:
Nice, but we should also have a test with just order on its own to ensure these can work together but also separately.